.retireignore

```diff
@@ -1,2 +1,11 @@
 node_modules/marked
-node_modules/jsdoc/node_modules/marked
\ No newline at end of file
+node_modules/jsdoc
+node_modules/marked
+node_modules/nsp
+node_modules/jsdoc
+node_modules/nodesecurity-npm-utils
+node_modules/silent-npm-registry-client
+node_modules/npm-registry-client
+node_modules/request
+node_modules/tough-cookie
+node_modules/map-stream
```

Comment:
@ksiegel I removed the unnecessary ignores. But some are still needed. Please review now.